File: Python-Task-05/Python Files/statistics_tool.py
numbers=[]

print("Enter 10 numbers:")

for i in range(10):
    numbers.append(int(input()))

largest=max(numbers)
smallest=min(numbers)
total=sum(numbers)
average=total/10

even=0
odd=0

for num in numbers:
    if num%2==0:
        even+=1
    else:
        odd+=1

print("\nLargest:",largest)
print("Smallest:",smallest)
print("Sum:",total)
print("Average:",average)
print("Even Numbers:",even)
print("Odd Numbers:",odd)

File: Python-Task-05/C Files/statistics_tool.c
#include<stdio.h>

int main(){

int num[10],i;
int largest,smallest,sum=0;
int even=0,odd=0;

printf("Enter 10 numbers:\n");

for(i=0;i<10;i++)
scanf("%d",&num[i]);

largest=smallest=num[0];

for(i=0;i<10;i++){

if(num[i]>largest)
largest=num[i];

if(num[i]<smallest)
smallest=num[i];

sum+=num[i];

if(num[i]%2==0)
even++;
else
odd++;

}

printf("\nLargest Number: %d",largest);
printf("\nSmallest Number: %d",smallest);
printf("\nSum: %d",sum);
printf("\nAverage: %.2f",(float)sum/10);
printf("\nEven Numbers: %d",even);
printf("\nOdd Numbers: %d",odd);

return 0;
}